packages/devtools_app/lib/src/shared/preferences.dart

```diff
@@ -150,7 +154,14 @@ class InspectorPreferencesController extends DisposableController
   final _pubRootDirectories = ListValueNotifier<String>([]);
   final _pubRootDirectoriesAreBusy = ValueNotifier<bool>(false);
   final _busyCounter = ValueNotifier<int>(0);
+  final _defaultInspectorDetailsView =
+      ValueNotifier<InspectorDetailsViewOptions>(
+    InspectorDetailsViewOptions.layoutExplorer,
+  );
+
   static const _hoverEvalModeStorageId = 'inspector.hoverEvalMode';
+  static const _defaultInspectorDetailsViewStorageId =
+      'inspector.detailsViewSelection';
```

Comment:
nit: let's name "inspector.defaultDetailsViewType" so that it is clear that this isn't updated each time they select a tab, but rather indicates their selection of a setting